guides/release/models/finding-records.md

```diff
@@ -82,43 +85,43 @@ For example, if your server API provides an endpoint for the currently logged in
 }
 ```
 
-And if the adapter for the `User` model defines a `queryRecord()` method that targets that endpoint:
-
-```javascript {data-filename=app/adapters/user.js}
-import Adapter from '@ember-data/adapter';
-import fetch from 'fetch';
-
-export default class UserAdapter extends Adapter {
-  queryRecord(store, type, query) {
-    return fetch('/api/current_user');
-  }
+And if the builders for the `User` model defines a `queryData()` method that targets that endpoint:
+
+```javascript {data-filename=app/builders/user.js}
+export function queryData() {
+    return {
+        url: `/api/current_user`,
```

Comment:
I feel like the approach here to use custom builder is correct one, but this exact builder implementation missing some key options in return. Let me find more info for you